src/components/Bag/downloads/index.js

```diff
@@ -55,12 +61,38 @@ const DownloadLinksDialog = ({
         onClose();
     };
 
+    const handleCopy = (event) => {
+        event.preventDefault();
+        event.stopPropagation();
+
+        const linksString = paths.reduce((acc, curr) => {
+            const fullLink = `${getHost()}/api/download?path=${curr}`;
+            if (acc !== "") {
+                acc = `${acc}\n${fullLink}`;
+            } else {
+                acc = fullLink;
+            }
+            return acc;
+        }, "");
+
+        navigator.clipboard
```

Comment:
For anyone else, I was curious about the difference between the copy method we're using [here](https://github.com/cyverse-de/ui-lib/blob/master/src/components/copy/clipboardFunctions.js) vs. this one.  Seems it's totally up to you which way you want to go with according to [this](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard)